Nodejs/Product/Nodejs/SharedProject/ProjectNode.cs

```diff
@@ -6473,6 +6473,14 @@ internal void OnItemAdded(HierarchyNode parent, HierarchyNode child, HierarchyNo
 
             var prev = previousVisible ?? child.PreviousVisibleSibling;
             var prevId = (prev != null) ? prev.HierarchyId : VSConstants.VSITEMID_NIL;
+
+            if (string.IsNullOrWhiteSpace(parent.GetItemName()))
```

Comment:
I'd be happier checking the HRESULT - I'm worried that null and empty might be valid values.  We're just trying to check for the absence of a node.

We don't have a way to simulate the failure?